app/src/test/java/io/apicurio/registry/DataUpgradeTest.java

```diff
@@ -0,0 +1,29 @@
+package io.apicurio.registry;
+
+import io.apicurio.registry.utils.tests.ApicurioTestTags;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+@QuarkusTest
+@TestProfile(DataUpgradeTestProfile.class)
+@Tag(ApicurioTestTags.SLOW)
+public class DataUpgradeTest extends AbstractResourceTestBase {
+
+    @Override
+    @BeforeEach
+    protected void beforeEach() throws Exception {
+        setupRestAssured();
+    }
+
+    @Test
+    public void testCheckImportedData() throws Exception {
```

Comment:
Could probably do with a few more checks.  :)

Reply:
Yup, I was just checking that the number of versions is the expected one, I'll add a *lot* more checks.